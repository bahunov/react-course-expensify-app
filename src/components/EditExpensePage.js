import React from 'react';
import {NavLink} from 'react-router-dom';

const EditExpensePage = () => {
    let id1 = 1
    let id2 = 2
    let id3 = 3
    return (
    <div>
        Edit page...!!!
        <NavLink to={'edit/'+id1} activeClassName="is-active">Edit Page {id1} </NavLink>
    </div>
    );
}

export default EditExpensePage;