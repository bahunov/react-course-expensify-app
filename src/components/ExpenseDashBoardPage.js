import React from 'react';
import ExepnseList from './ExpenseList';
import ExpenseListFilter from './ExpenseListFilter';
import ExpensesSummary from './ExpensesSummary';


const ExpenseDashBoardPage = () => (
    <div>
    <ExpensesSummary />
    <ExpenseListFilter />
    <ExepnseList/>
    </div>
); 

export default ExpenseDashBoardPage;