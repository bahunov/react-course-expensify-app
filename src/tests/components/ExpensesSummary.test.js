import React from 'react';
import { shallow } from 'enzyme';
import { ExpensesSummary } from '../../components/ExpensesSummary';

test('should render expensesSummary with 1 expense' , () =>{
    const wrapper = shallow(<ExpensesSummary expenseCount={1} expenseTotal={235} />)
    expect(wrapper).toMatchSnapshot();
});

test('should render expensesSummary with multiple expense' , () =>{
    const wrapper = shallow(<ExpensesSummary expenseCount={1} expenseTotal={232342345} />)
    expect(wrapper).toMatchSnapshot();
});