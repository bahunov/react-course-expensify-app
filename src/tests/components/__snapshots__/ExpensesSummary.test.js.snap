// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render expensesSummary with 1 expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totaling 
    $2.35
  </h1>
</div>
`;

exports[`should render expensesSummary with multiple expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totaling 
    $2,323,423.45
  </h1>
</div>
`;
